components/NavBrand.js

```diff
@@ -13,7 +13,8 @@ const NavBrandContainer = styled.div`
         margin-left: auto;
     }
     @media only screen and (max-width: ${(props) => props.theme.breakpoints.sm}) {
-        margin-left: ${(props) => props.theme.spaces.md}
+        margin-left: ${(props) => props.theme.spaces.md};
+        width: ${(props) => props.theme.spaces.xxxl.replace("rem","") * 0.7}rem;
```

Comment:
This is fine but I also don't think there's anything wrong with using a hard coded width number for elements like this. I think of spacing more as used for margin/padding etc.

Reply:
Fair enough.
Will change line 17 to:
`width: 6.72rem;`